tests/cli/test_cli_options.py

```diff
@@ -631,7 +636,7 @@ def main(args):
             gen_expected_path=s_path_join(sc_dir, 'output/universalindent.cfg'),
             gen_result_path=s_path_join(sc_dir, 'results/universalindent.cfg'),
             gen_result_manip=[reg_replace(r'version=U.+', ''),
-                              reg_replace(r'\([0-9]+\)', '')]
+                              reg_replace('\(\d+\)', '')]
```

Comment:
Um... this does seem to work by accident, but I think you should keep this as a raw string...